src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -289,19 +289,34 @@ interface IOptionSettlementEngine {
     //  Accessors
     //////////////////////////////////////////////////////////////*/
 
+    //
+    // Option Information
+    //
+
+    /**
+     * @notice Gets option type information for a given tokenId.
+     * @param optionId The tokenId of the option.
+     * @return optionInfo The Option struct for tokenId if the optionKey for tokenId is initialized.
```

Comment:
I'd just say "exists" rather than is initialized.

Reply:
Actually this may be superfluous — we throw if TokenNotFound, think that's sufficient

Reply:
Cleaned up both option() claim() underlying() in next commit